reference/twig_reference.rst

```diff
@@ -302,15 +302,19 @@ is_granted
     {{ is_granted(role, object = null, field = null) }}
 
 ``role``
-    **type**: ``string``
+    **type**: ``string``, ``string[]``
 ``object`` *(optional)*
     **type**: ``object``
 ``field`` *(optional)*
     **type**: ``string``
 
-Returns ``true`` if the current user has the required role. Optionally,
-an object can be pasted to be used by the voter. More information can be
-found in :ref:`security-template`.
+Returns ``true`` if the current user has the required role, if only one 
+is passed; if more than one is passed, with an array, the behavior depends
```

Comment:
I don't see how this behaviour *could* be configured.

Reply:
It depends on the strategy you're using: 
 * `affirmative` (default) grants access as soon as there is one voter granting access;
 * `consensus` grants access if there are more voters granting access than there are denying;
 * `unanimous` only grants access if none of the voters has denied access;

See https://symfony.com/doc/current/components/security/authorization.html#access-decision-manager

Reply:
The access decision strategy defines how the result looks like when multiple voters voted on a certain attribute. It does not affect the internal behaviour of the `RoleVoter`.

Reply:
see #10580

Reply:
Umh you're right, I've mistakenly confused the two things. Thanks for spotting it.